packages/atlas/src/api/queries/notifications.graphql

```diff
@@ -1,121 +1,164 @@
+            }
+            ... on VideoFeaturedOnCategoryPage {
+              categoryId
+              categoryName
+              videoTitle
+            }
+            ... on NftFeaturedOnMarketPlace {
+              videoId
+              videoTitle
+            }
+            ... on VideoFeaturedAsCategoryHero {
+              categoryId
+              categoryName
+              videoTitle
+            }
+            ... on VideoExcluded {
+              videoTitle
+            }
+            ... on NewChannelFollower {
+              followerHandle
+            }
+            ... on CommentPostedToVideo {
+              memberHandle
+              videoId
+              videoTitle
+            }
+            ... on VideoLiked {
+              videoId
+              videoTitle
+            }
+            ... on VideoDisliked {
+              videoId
+              videoTitle
+            }
+            ... on ChannelVerified {
+              __typename
+            }
+            ... on ChannelSuspended {
+              __typename
+            }
+            ... on NftPurchased {
+              buyerHandle
               price
-              nft {
-                video {
-                  ...BasicVideoActivityFields
-                }
-              }
+              videoTitle
+              videoId
             }
-
-            ... on BidMadeCompletingAuctionEventData {
-              winningBid {
-                bidder {
-                  ...BasicMembershipFields
-                }
-                amount
-                nft {
-                  video {
-                    ...BasicVideoActivityFields
-                  }
-                }
-              }
-              previousNftOwner {
-                ...BasicNftOwnerFields
-              }
+            ... on CreatorReceivesAuctionBid {
+              amount
+              bidderHandle
+              videoId
+              videoTitle
             }
-
-            ... on OpenAuctionBidAcceptedEventData {
-              winningBid {
-                amount
-                bidder {
-                  ...BasicMembershipFields
-                }
-                nft {
-                  video {
-                    ...BasicVideoActivityFields
-                  }
-                }
-              }
-              previousNftOwner {
-                ...BasicNftOwnerFields
-              }
+            ... on NftRoyaltyPaid {
+              amount
+              videoId
+              videoTitle
             }
-
-            ... on EnglishAuctionSettledEventData {
-              winningBid {
-                bidder {
-                  ...BasicMembershipFields
-                }
-                nft {
-                  video {
-                    ...BasicVideoActivityFields
-                  }
-                }
-              }
-              previousNftOwner {
-                ...BasicNftOwnerFields
-              }
+            ... on EnglishAuctionSettled {
```

Comment:
I believe we should deprecate this.
The design specifies a notification for timed Auction expired, but technically is not this.
Bidders will be notified in any case when an english (= timed) auction is settled with a `AuctionWon` or `AuctionLost` notification